engine/src/test/java/org/pentaho/di/trans/steps/rest/RestTest.java

```diff
@@ -77,6 +75,7 @@ public void testCreateMultivalueMap() {
     assertTrue( val1.contains( "%7D" ) );
   }
 
+  @Ignore
```

Comment:
Why are we ignoring this test now?

Reply:
Based on the comments left in the ticket, Apache Hop project was a fork from Pentaho.
They solved the same bug, but they move further to upgrade underlying libraries from 1.x to 2.x (jersey-client, etc), which we don't plan to do at this moment.

This PR **just** bring in changes in Rest.java. no library changes at all.

I got the idea to reference test code from Apache Hop, this test was marked `@Ignore`, so I assumed we can ignore for now, maybe wait till we do library upgrade and revisit it.